itest/tests/contextMenu.test.js

```diff
@@ -34,24 +61,63 @@ describe("Test search mods via right-clicks", () => {
     }
   })
 
-  beforeEach(async () => {
-    await writeSearch(app, "")
-    await startSearch(app)
-    await waitForResults(app)
+  stdTest("FilterEq ensure ingest", (done) => {
+    writeSearch(app, `* | count() by _path | sort _path`)
+      .then(async () => {
+        await startSearch(app)
+        expect(await searchDisplay(app)).toMatchSnapshot()
+        done()
+      })
+      .catch((err) => {
+        handleError(app, err, done)
+      })
   })
 
-  stdTest("count by _path; sort asc", (done) => {
-    rightClick(app, selectors.viewer.resultCellContaining("capture_loss"))
+  const run = (path: string, values: string[]) => {
+    let testIdx = 0
+    let testId: string
```

Comment:
I think this declaration can be moved to line 81. I don't a see a need to declare testId here, then assign it down there. I may be missing something though.

Reply:
Good spot. Leftover from an internal refactor. Fixed.